src/permission/permission.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { PermissionGroup } from 'src/permission-group/permission-group.model';
+import { PermissionController } from './permission.controller';
+import { Permission } from './permission.model';
+import { PermissionService } from './permission.service';
+
+@Module({
+    controllers: [PermissionController],
+    providers: [PermissionService],
+    imports: [SequelizeModule.forFeature([Permission, PermissionGroup])],
```

Comment:
## create export for permissions and import them into permission group. create functions to CRUD permissions at permission group